src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -148,20 +151,68 @@ bool VisitAggregateAttribute(ITypeSymbol _attributeSymbol,int depth)
 			}
 		}
 
-		public bool IsBoundAttribute(ITypeSymbol attributeSymbol)
+		public BoundAttribute IsBoundAttribute(AttributeData attribute)
 		{
-			foreach (var baseType in BoundBaseTypes)
+            foreach (var baseType in BoundBaseTypes)
 			{
-				if (AttributeDerivedFromClass(attributeSymbol, baseType))
-					return true;
+				if (AttributeDerivedFromClass(attribute.AttributeClass, baseType))
+					return BoundAttribute.DbBound;
 			}
-			return false;
+            if (attribute.AttributeClass.GetMembers().Select(a => a.Name).Contains(_IsDBField))
+            {
+                foreach (var argument in attribute.NamedArguments)
+                {
+                    if (argument.Key.Equals(_IsDBField))
+                    {
+                        if (argument.Value.Value.Equals(true))
+                            return BoundAttribute.DbBound;
+                        else
+                            return BoundAttribute.Unbound;
+                    }
+                        
+                }
+                return BoundAttribute.Unknown;
+            }
+            return BoundAttribute.Unbound;
 		}
-		
-		public bool ContainsBoundAttributes(IEnumerable<ITypeSymbol> attributesSymbols)
-		{
-			return attributesSymbols.Any(IsBoundAttribute);
+
+        ///TODO: refactoring arguments -> remove semanticModel to constructor? Is it nessesary.
+        ///TODO: Add DataFlow analyze to corner cases with defaul IsDBBound assigment.
+        public bool? IsBoundField(PropertyDeclarationSyntax property,SemanticModel semanticModel)
+        {
+            var typeSymbol = semanticModel.GetDeclaredSymbol(property);
+            var attributesData = typeSymbol.GetAttributes();
+
+            foreach(var attribute in attributesData)
+            {
+                if (IsBoundAttribute(attribute) == BoundAttribute.DbBound)
+                    return true;
+                foreach(var argument in attribute.NamedArguments)
+                {
+                    if(argument.Key.Equals("IsDBField") && argument.Value.Value.Equals(true))
+                        return (bool) argument.Value.Value;
+                }    
+            }
+            return false; 
+        }
+
+        public BoundAttribute ContainsBoundAttributes(IEnumerable<AttributeData> attributes)
+        {
+            foreach (var attribute in attributes)
+            {
+                BoundAttribute result = IsBoundAttribute(attribute);
+                if (result == BoundAttribute.DbBound || result == BoundAttribute.Unknown)
+                    return result;
+            }
+			return BoundAttribute.Unbound;
 		}
 
 	}
+
+    public enum BoundAttribute
```

Comment:
Bad naming. `Attribute` postfix in C# is used for the classes derived from `Attribute`.